Mathlib/Topology/Spectral/Hom.lean

```diff
@@ -33,6 +34,7 @@ variable [TopologicalSpace α] [TopologicalSpace β] [TopologicalSpace γ] {f :
 
 /-- A function between topological spaces is spectral if it is continuous and the preimage of every
 compact open set is compact open. -/
+@[stacks 005A, stacks 08YG]
```

Comment:
```suggestion
@[stacks [005A, 08YG]]
```
untested, but I'm curious if this works. Or if there's some other way of combining these - I vaguely recall one existing?

Reply:
Already tried this syntax and a few variants. Couldn't get anything to work

Reply:
I only noticed this now: as you confirmed, multiple stacks tags can only be applied repeating `stacks`.  This could of course be changed, but I have the impression that there will be very few results in mathlib that cover more than one stacks tag and I would expect the opposite to be true more often!